### PR TITLE
Improve CI

### PR DESCRIPTION
**Pull request by haya14busa**

### CODE REVIEW

File: .drone.sec [hunk 1]
```diff
@@ -1 +1 @@
-eyJhbGciOiJSU0EtT0FFUCIsImVuYyI6IkExMjhHQ00ifQ.P7gO-DzPY8QSCfyD2kXKnsIwn_qLr-fmwFsCGBowZLukKd2e_CVKv_FHaWnDNTVk-L-mmR5e6e5ztc2Gxmnow50FJmaaC4Gq3FYzYteLa7VFWXXMD9OzkEMKOxvD8YUXVZUBlqKioR6xDBsTTPJGOXvJs6TLf_e96Iuac02LaxcP0XgrWzdQZkWKzcxhv06SIciEPk7pPVZHly8UMCo45AHEYJ-aaSlaFuLEQjt0fSiFJWbgF1QUmyhey43CYaau6S66EkIz8N73HQsqyExc0dK5_vJu8DGwDLzCVopHGb1ayuixFe8Xzrg-5a-Cfu32VpHGmBmXR6n1YmZmKz_oYA.eLr96Sle2G3P2PmU.juxqdo9S7UWritzidPeVLMUqXfuN1lJEU5Ewzli7mcbfC9PSgBRl21y6lO_oVssRyQZjwqhZHEr9flChO3RjZrx-525kzjlAUzxON1a77ukcTz3FAMc_uH3xy5bscg7az79-ExnzMkkm1b4_iA0vGHIg3cI5LdgbUNkDoUTkqk-GjDGd5-_oPkB1_CjULI6ViN7goHxc5QaWEFvZoX3c.nWImV_uG1pVdqJtozCMLxg
```
Comment (haya14busa) [inline .drone.sec]: test body

Comment (reviewdog) [inline .drone.sec]: test body

File: reviewdog.go [hunk 2]
```diff
@@ -70,7 +71,11 @@ type DiffService interface {
 
 // Run runs Reviewdog application.
 func (w *Reviewdog) Run(r io.Reader) error {
-	results, err := w.p.Parse(r)
+
+	b, _ := ioutil.ReadAll(r)
```
Comment (reviewdog) [inline reviewdog.go]: **[errcheck]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
	b, _ := ioutil.ReadAll(r)

File: github.go [hunk 3]
```diff
@@ -68,6 +69,15 @@ func NewGitHubPullReqest(cli *github.Client, owner, repo string, pr int, sha str
 	}
 }
 
+func F() {
+	go func() {
+		var tick = time.Tick(1 * time.Second)
```
Comment (reviewdog) [inline github.go]: **[staticcheck]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
using time.Tick leaks the underlying ticker, consider using it only in endless functions, tests and the main package, and use time.NewTicker here (SA1015)

**Pull request merged**